libcxx/include/__type_traits/has_unique_object_representation.h

```diff
@@ -24,7 +22,7 @@ _LIBCPP_BEGIN_NAMESPACE_STD
 
 template <class _Tp>
 struct _LIBCPP_TEMPLATE_VIS has_unique_object_representations
-    : public integral_constant<bool, __has_unique_object_representations(remove_cv_t<remove_all_extents_t<_Tp>>)> {};
+    : public integral_constant<bool, __has_unique_object_representations(_Tp)> {};
 
 template <class _Tp>
 inline constexpr bool has_unique_object_representations_v = has_unique_object_representations<_Tp>::value;
```

Comment:
```suggestion
inline constexpr bool has_unique_object_representations_v = __has_unique_object_representations(_Tp);
```
Might as well while we're here.

Reply:
**Very** happy to do this if we're okay with doing it elsewhere (I can lead the clean-up for that).

Reply:
Depends on what you mean by "it". If you just mean cases where we always have the builtins available, sure (although I think I removed those cases). In other cases I'm not sure it's a good idea.

Reply:
I mean applying the same change to all other traits where the trait is how the builtin is surfaced (so mostly stuff in `<type_traits>`). Anything else deserves a proper discussion.

Reply:
I think what Nikolas tried explaining is that in most cases, we already call the builtin directly from the `_v` alias if there's a builtin (for example `libcxx/include/__type_traits/is_reference.h`). So I think there should be no other places where we want to make the same change, unless of course we forgot to do it in other places.